parla/task_runtime.py

```diff
@@ -141,7 +141,8 @@ def clear_dependencies(self):
 
     def __repr__(self):
         if self.func:
-            return "TaskRunning({}, {}, {})".format(self.func.__name__, self.args, self.dependencies)
```

Comment:
Why should you remove this?

Reply:
It printed so much information that it made debugging too messy for me haha. I've left the original version commented there in case anyone needs it later. If you'd like, I can revert it.

Reply:
no problem. it's fine for me.